### PR TITLE
DOC-1839 replace kubectl with oc

### PR DESCRIPTION
**Pull request by kaitlynmichael**
DOC-1839
Replace kubectl commands in the OpenShift CLI guide with the oc command. 
Staged preview: https://docs.redis.com/staging/DOC-1839/kubernetes/deployment/openshift/openshift-cli/

### CODE REVIEW

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 1]
```diff
@@ -182,7 +182,8 @@ Each Redis Enterprise cluster requires at least 3 nodes. Single-node RECs are no
 1. Patch the validating webhook with the certificate.
 
     ```sh
-    kubectl patch ValidatingWebhookConfiguration redis-enterprise-admission --patch "$(cat modified-webhook.yaml)"
+    oc patch ValidatingWebhookConfiguration /
+    redis-enterprise-admission --patch "$(cat modified-webhook.yaml)"
```
Comment (zcahana) [inline content/kubernetes/deployment/openshift/openshift-cli.md:186]: 1. Use a backslash (and not a forward slash) to split a command over multiple lines.
2. Multi line commands are typically indented from the 2nd line and on.
3. Repeat over additional occurrences below.
```suggestion
    oc patch ValidatingWebhookConfiguration \
        redis-enterprise-admission --patch "$(cat modified-webhook.yaml)"
```

**Review by zcahana — CHANGES_REQUESTED** [review]
See one inline comment.

Note that the prerequisites section mentions both `kubectl` and `oc`. `kubectl` can now be removed from the prerequisites.

Also, `oc` links to an old OpenShit 4.8 page. This can be updated to the current latest:
https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html

We can also use `latest` in the URL, which gets redirected to the current latest version:
https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html

...but there's no guarantee that this URL will remain valid in future versions which might change the specific page location:

File: content/kubernetes/deployment/openshift/openshift-cli.md (hunk 1 above)
Comment (kaitlynmichael) [reply content/kubernetes/deployment/openshift/openshift-cli.md:186]: @zcahana if I indent the second line, will it still work to copy and paste into the terminal?

**Review by cmilesb — APPROVED** [review]
Looks good to me!

File: content/kubernetes/deployment/openshift/openshift-cli.md (hunk 1 above)
Comment (zcahana) [reply content/kubernetes/deployment/openshift/openshift-cli.md]: Definitely.

**Review by zcahana — APPROVED** [review]
Approved but please also consider [this comment](https://github.com/RedisLabs/redislabs-docs/pull/2520#pullrequestreview-1403257693) which is nice to have.

**Pull request merged**